README.md

```diff
@@ -0,0 +1,20 @@
+What is this?
+=============
+
+The FiftyTwo Activity is a simple cards game. It support mesh play with up to four players (for games like Hearts or Spades), and it already supports single-person play against the computer (basic AI play), and multi-player (on the same computer) play. .
```

Comment:
**_(basic AI play)_** is not necessary as there's really no AI.

Reply:
I agree with you but I had found this in one Sugar documentation. Hence planned to follow the documentation and update readme accordingly.